backend/core/views.py

```
@@ -147,17 +149,89 @@
 MODULE_PATHS = SETTINGS_MODULE.settings.MODULE_PATHS
 
 
+class MultipleCharFilter(df.MultipleChoiceFilter):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)